src/project/internal/notationproject.cpp

```diff
@@ -522,20 +522,24 @@ mu::Ret NotationProject::saveSelectionOnScore(const mu::io::path& path)
     // Write project
     std::string suffix = io::suffix(info.fileName());
     MscWriter::Params params;
+    QString org_path = m_engravingProject->path();
```

Comment:
I don't think we're using snake_case variable names, try to stick to camelCase.  And personally I'd call it originalPath.  But it's not clear that you *should* have to change the path property for the project just to be able to save to a different filename - can't you just change params.filePath?

Reply:
here's our code style info https://github.com/musescore/MuseScore/wiki/CodeGuidelines